res/linux/69-mixxx-usb-uaccess.rules

```diff
@@ -10,8 +10,6 @@
 
 # New IDs must be also added to res/linux/mixxx.metainfo.xml
 
-# Install and execute before 70-uaccess.rules, e.g. .../udev/rules.d/69-mixxx-usb-uaccess.rules
```

Comment:
Please leave the comment.

Reply:
Also, we now have a `packaging` directory in the repository. Could you move this from `res/linux` to `packaging/linux`?

Reply:
Done and done